server/src/main/java/com/cloud/network/vpc/NetworkACLServiceImpl.java

```diff
@@ -1146,6 +1146,9 @@ protected void validateMoveAclRulesData(NetworkACLItemVO ruleBeingMoved, Network
         NetworkACLVO acl = _networkACLDao.findById(aclId);
         Vpc vpc = _entityMgr.findById(Vpc.class, acl.getVpcId());
         Account caller = CallContext.current().getCallingAccount();
+        if (acl.getVpcId() == null) {
+            throw new InvalidParameterValueException("Unable to move rules for a default ACL");
+        }
```

Comment:
@Pearl1594 
What does this change fix ?

Reply:
Ah sorry - this wasn't to be part of this PR - this is ideally to prevent an NPE that happens when we try re-ordering rules for a default acl

Reply:
good .. no problem